projects/js-packages/components/components/icons/index.jsx

```diff
@@ -26,20 +27,22 @@ function IconWrapper( {
 	size = 24,
 	viewBox = '0 0 24 24',
 	opacity = 1,
-	color = '#2C3338',
+	color,
```

Comment:
@pablinos Removing the default probably will break some current usages, not? Any reason to do it?

Reply:
With the colour prop it sets the `fill` attribute and makes overriding it difficult, so instead [I set the default colour in the CSS](https://github.com/Automattic/jetpack/pull/23795/files#diff-60b2cde8f5df00ff59380f078572b8ec302053a978eaa840c152ee1567d36a01R4)